robonomics_liability/src/robonomics_liability/executor.py

```diff
@@ -26,7 +27,9 @@ def __init__(self):
 
         web3_provider = rospy.get_param('~web3_http_provider')
         self.web3 = Web3(HTTPProvider(web3_provider))
-        self.account = self.web3.eth.accounts[0]
+
+        self.account = rospy.get_param('~eth_account_address')
+        self.account = self.web3.eth.accounts[0] if len(self.account) == 0 else self.account
```

Comment:
Just use get_param second argument, it used if param isn’t set.

Reply:
I think, this will not work for us since in the current architecture we have launch file, where we should define eth_account_address argument. 
In this case, it should either have a default value or value should be set by launch file user. Both requirements do not work: in the first case roslaunch should somehow fetch the 0-th web3 account, in the second case we break backward compatibility and force user to set the address.